readthedocs/core/resolver.py

```diff
@@ -71,10 +72,8 @@ def _fix_filename(project, filename):
 def base_resolve_path(project_slug, filename, version_slug=None, language=None, private=False,
                       single_version=None, subproject_slug=None,  subdomain=None, cname=None):
     """ Resolve a with nothing smart, just filling in the blanks."""
-
-    if private:
-        url = '/docs/{project_slug}/'
-    elif subdomain or cname:
+    use_subdomain = getattr(settings, 'USE_SUBDOMAIN', False)
+    if subdomain or cname or (private and use_subdomain):
         url = '/'
     else:
         url = '/docs/{project_slug}/'
```

Comment:
Seems like maybe we switch the logic here. Seems that `url = '/'` is the default, and `url = '/docs/{project_slug}/'` is the special case. Is this only hit in dev?